content/docs-v1/pages.md

```diff
@@ -101,45 +100,55 @@ In the following example publication outline, we’ve listed the files and direc
+
+Like in the case of sub-sections explained above, `index.md` files always inherit the URL of their parent directory. The `index.md` file used for your cover is in the root, or top-most, directory, and so the URL for it will be the base URL where you host the site.
 
 ## Hide/Show Pages
 
-By default, every page you create will be included in all formats of your publication (online, PDF/print, and e-book). Every page will also automatically be listed in the publication’s menu and contents pages. However, this can be overridden by setting any of the following Page YAML attributes to `false`.
+By default, every page you create will be included in all formats of your publication (html, PDF/print, and e-book). This can be overridden by including an `outputs` attribute and excluding the undesired formats from the array (`epub`, `pdf`, `html`). For example, if you want your Copyright page to appear in the PDF and EPUB formats but not in the online version, you would only list `epub` and `pdf` as page YAML values.
+
+```YAML
+outputs:
+  - epub
+  - pdf
+```
+Likewise, if you wanted to include the About page in the online version and exclude it from the PDF and EPUB formats, you would only list `html`.
+
+```yaml
+outputs:
+  - html
+```
+
+Every page will also automatically be listed in the publication’s menu and contents pages. However, this can be overridden by setting any of the following page YAML attributes to `false`.
 
 ```yaml
 toc:
 menu:
-online:
-pdf:
-epub:
 ```
 
-This allows you to do things like including an About page in your online edition, but a more traditional Copyright page in print. Or to substitute a simple splash page as a section break in the print, for the more elaborate contents grid you might use online.
-
 {{< q-class "box tip" >}}
-- Note that when setting `online: false`, the page will not be included in the linear ordering of the book or in the menu, table of contents, or search index, but it is still built. When deploying your site from the built files in the `/site/` directory, simply delete any unneeded ones. Read more about site deployment in the [*Deploy Your Project*](/docs-v1/site-deploy/) section of this guide.
+- Note that when omitting `online` from `outputs` the page will not be included in the linear ordering of the book or in the menu, table of contents, or search index, but it is still built. When deploying your site from the built files simply delete any unneeded ones. Read more about site deployment in the [*Deploy Your Project*](/docs-v1/site-deploy/) section of this guide.
```

Comment:
One more instance of `online` instead of `html` here. Do you also want to do a search for any others that might be hiding in other parts of this file or elsewhere?

Reply:
Ohhh I didn't realize it applied to that attribute too. Ok, yes, I will search.

Reply:
Oh wait ignore that comment. I keep getting confused. This was a real brain teaser for me today for some reason.